connectivity/FEATURE_BLE/source/generic/GapImpl.cpp

```diff
@@ -2985,7 +3278,24 @@ void Gap::on_remote_connection_parameter(
 #if BLE_ROLE_OBSERVER
 ble_error_t Gap::setScanParameters(const ScanParameters &params)
 {
+    tr_info("Set scan parameters - "
+            "own_address_type=%s, "
+            "scanning_filter_policy=%s, "
+            "phys=%d ,"
```

Comment:
maybe instead of printing this enum as integer just print the phys that are being configured
if (phy1) tr_info("phy1 config")
if (phy coded) tr_info("coded config")